src/main/java/org/icgc/argo/workflow_management/util/Reflections.java

```diff
@@ -42,21 +62,17 @@
 
       try {
         if (nonNull(args)) {
-          method.invoke(obj, args);
+          return (U) method.invoke(obj, args);
         } else {
-          method.invoke(obj);
+          return (U) method.invoke(obj);
         }
       } catch (IllegalAccessException | InvocationTargetException e) {
         log.error(String.format("invoke error for method: %s", methodName), e);
       }
     } else {
       throw new ReflectionUtilsException(String.format("Cannot access method: %s", methodName));
     }
-  }
-
-  public static <T> void invokeDeclaredMethod(T obj, String methodName)
-      throws ReflectionUtilsException {
-    invokeDeclaredMethod(obj, methodName, null);
+    return null;
```

Comment:
once ReflectionUtilsException is used (described in previous comment), there should be no need for this return, as it cannot be reached.